### PR TITLE
refactor: upgrade to Dart 2.19

### PR DESCRIPTION
**Pull request by felangel**
## Description

- refactor: upgrade to Dart 2.19

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [X] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: .github/workflows/flutter_news_example.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
 
     strategy:
       matrix:
-        channel: [stable, beta]
```
Comment (domesticmouse) [inline .github/workflows/flutter_news_example.yaml:31]: A new `beta` is about to drop, i'd suggest not making this change.

**Review by domesticmouse — CHANGES_REQUESTED** [review]

File: .github/workflows/flutter_news_example.yaml (hunk 1 above)
Comment (felangel) [reply .github/workflows/flutter_news_example.yaml:31]: Was just verifying the other jobs are passing. I don’t intend to merge this change 👍

**Comment by felangel** [comment]
@domesticmouse it appears the stable channel requires intl 0.17.0 whereas the beta channel requires intl 0.18.0 so we're unable to satisfy both constraints. What do you suggest in this case?

**Comment by domesticmouse** [comment]
Ugh. Comment out beta, and create a PR that re-enables beta with the fixes required to make `beta` happy. You'll need that PR in a couple of months 😏

**Comment by felangel** [comment]
> Ugh. Comment out beta, and create a PR that re-enables beta with the fixes required to make `beta` happy. You'll need that PR in a couple of months 😏

Sounds good 👍

**Comment by felangel** [comment]
@domesticmouse whenever you have a min can you take another look? Thanks!

**Review by domesticmouse — APPROVED** [review]

**Review by kaiceyd — APPROVED** [review]

**Pull request merged**